packages/composites/historical-average/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@chainlink/historical-average-adapter",
+  "version": "0.0.0",
```

Should we start with 1.0.0?
```suggestion
  "version": "1.0.0",
```

---

The changeset should bump this to 1.0.0

---

Got it